app/src/main/java/org/sopt/and/ui/screen/signIn/composable/SignInScreen.kt

```diff
@@ -0,0 +1,256 @@
+package org.sopt.and.ui.screen.signIn.composable
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material3.AlertDialog
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.hilt.navigation.compose.hiltViewModel
+import org.sopt.and.R
+import org.sopt.and.ui.components.textField.TextFieldCustom
+import org.sopt.and.ui.screen.signIn.viewmodel.SignInViewModel
+import org.sopt.and.ui.theme.BlueBtnColor
+import org.sopt.and.ui.theme.darkGray1
+import org.sopt.and.ui.theme.darkGray3
+
+
+@Composable
+fun SignInScreen(
+    navController: androidx.navigation.NavHostController,
+    modifier: Modifier,
+    viewModel: SignInViewModel = hiltViewModel()
+) {
+    val context = LocalContext.current
+    val loginDescription = stringResource(id = R.string.login_description)
+    Box(modifier = modifier) {
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .background(darkGray1)
+        ) {
+            Spacer(modifier = Modifier.height(30.dp))
+            TextFieldCustom(
+                value = viewModel.userIdInput,
+                placeholder = "아이디 주소 또는 아이디",
+                onValueChange = { value ->
+                    viewModel.onUserIdInputChange(value)
+                },
```

Comment:
```suggestion
                onValueChange = viewModel::onUserIdInputChange,
```
람다의 매개변수가 동일하므로 이렇게 쓸 수 있어요. **(메서드 참조)**
메서드 참조를 이용하면 불안정한 값을 capture하는 람다에 의한 Recomposition을 줄일 수 있습니다 !

Reply:
덕분에 훨씬 코드가 간결해졌습니다. 좋은 정보 감사합니다!